src/Core/I18N/index.js

```diff
@@ -0,0 +1,47 @@
+const I18N_KEYS = {
+    APP_NAME: "APP_NAME",
+    SEARCH_TITLE: "SEARCH_TITLE",
+    SEARCH_PLACEHOLDER: "SEARCH_PLACEHOLDER",
+    SEARCH_BUTTON: "SEARCH_BUTTON",
+    SORT_BY: "SORT_BY",
+    ALL: "ALL",
+    RELEASE_DATE: "RELEASE_DATE",
+    MOVIE_ID: "MOVIE_ID",
+    MOVIE_RELEASE_DATE: "MOVIE_RELEASE_DATE",
+    MOVIE_TITLE: "MOVIE_TITLE",
+    MOVIE_URL: "MOVIE_URL",
+    MOVIE_WEBSITE: "MOVIE_WEBSITE",
+    MOVIE_OVERVIEW: "MOVIE_OVERVIEW",
+    MOVIE_RUNTIME: "MOVIE_RUNTIME",
+    ADD_MOVIE: "ADD_MOVIE",
+    EDIT_MOVIE: "EDIT_MOVIE",
+    NO_MOVIE_FOUND: "NO_MOVIE_FOUND",
+    DELETE_MOVIE: "DELETE_MOVIE",
+    DELETE_MOVIE_CONFIRM: "DELETE_MOVIE_CONFIRM",
+    TITLE: "TITLE",
+    GENRE: "GENRE",
+    RATING: "RATING",
+    OVERVIEW: "OVERVIEW",
+    RUNTIME: "RUNTIME",
+    RESET: "RESET",
+    SUBMIT: "SUBMIT",
+    CONFIRM: "CONFIRM",
+    EDIT: "EDIT",
+    DELETE: "DELETE", 
+    ERROR_TEXT: "ERROR_TEXT",
+    PAGE_NOT_FOUND: "PAGE_NOT_FOUND",
+    FOUR_OH_FOUR: "FOUR_OH_FOUR",
+    GO_HOME: "GO_HOME"
+}
+
+//Translation Files
+const { EN } = require("./en.js");
```

Comment:
Consider lifting up all imports to the top to follow:
Top - input
Middle - logic
Bottom - output

This way all your files will look the same which greatly improves readability.